### PR TITLE
Add NEAR stake, add key, delete key, create account, delete account

### PR DESCRIPTION
**Pull request by lamnd91**
## Description

Support Near actions: stake, add key, delete key, create account, delete account 

## How to test

unit test already write at:  tests/NEAR/SerializationTests.cpp

## Types of changes

* New feature (non-breaking change which adds functionality) 

## Checklist

- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.

### CODE REVIEW

File: src/NEAR/Serialization.cpp [hunk 1]
```diff
@@ -68,6 +118,21 @@ static void writeAction(Data& data, const Proto::Action& action) {
         case Proto::Action::kFunctionCall:
             writeFunctionCall(data, action.function_call());
             return;
+        case Proto::Action::kStake:
+            writeStake(data, action.stake());
+            return;
+        case Proto::Action::kAddKey:
+            writeAddKey(data, action.add_key());
+            break;
```
Comment (hewigovens) [inline src/NEAR/Serialization.cpp:126]: `return` and `break` usage are mixed in this `switch`, can you consolidate them?

File: src/NEAR/Serialization.cpp [hunk 2]
```diff
@@ -51,14 +51,64 @@ static void writeTransfer(Data& data, const Proto::Transfer& transfer) {
...
+    writePublicKey(data, addKey.public_key());
+    writeAccessKey(data, addKey.access_key());
+}
+
+static void writeDeleteKey(Data& data, const Proto::DeleteKey& deleteKey) {
+    writePublicKey(data, deleteKey.public_key());
+}
+
+static void writeCreateAccount(Data& data, const Proto::CreateAccount& createAccount) {
+}
```
Comment (hewigovens) [inline src/NEAR/Serialization.cpp:106]: no need to add this if it does nothing?

File: tests/NEAR/SerializationTests.cpp [hunk 3]
```diff
@@ -76,4 +76,173 @@ TEST(NEARSerialization, SerializeFunctionCallTransaction) {
...
+    auto blockHash = Base58::bitcoin.decode("244ZQ9cgj3CQ6bWBdytfrJMuMQ1jdXLFGnr4HhvtCTnM");
+    input.set_block_hash(blockHash.data(), blockHash.size());
+
+    auto privateKey = Base58::bitcoin.decode("3hoMW1HvnRLSFCLZnvPzWeoGwtdHzke34B2cTHM8rhcbG3TbuLKtShTv3DvyejnXKXKBiV7YPkLeqUHN1ghnqpFv");
+    input.set_private_key(privateKey.data(), 32);
+
+    auto serialized = transactionData(input);
+    auto serializedHex = hex(serialized);
+
+    ASSERT_EQ(serializedHex, "09000000746573742e6e65617200917b3d268d4b58f7fec1b150bd68d69be3ee5d4cc39855e341538465bb77860d01000000000000000d00000077686174657665722e6e6561720fa473fd26901df296be6adc4cc4df34d040efa2435224b6986910e630c2fef601000000040100000000000000000000000000000000917b3d268d4b58f7fec1b150bd68d69be3ee5d4cc39855e341538465bb77860d");
```
Comment (hewigovens) [inline tests/NEAR/SerializationTests.cpp:106]: do you have real world tx sent to mainnet?

**Review by hewigovens — APPROVED** [review]
could you please merge master? wasm ci is fixed, also could you please add a mainnet staking tx signed by wallet core? others LGTM

File: tests/NEAR/SerializationTests.cpp (hunk 3 above)
Comment (lamnd91) [reply tests/NEAR/SerializationTests.cpp:106]: i don't have a tx stake in mainnet. But i tried stake in testnet. Please take a look at this https://explorer.testnet.near.org/transactions/CMVgGzXZFkMb98tLq5zvti2DUc9yynZeY3kGPc234ZW6

Comment (hewigovens) [reply tests/NEAR/SerializationTests.cpp:106]: Could you please add this to test ?

**Pull request closed without merging**